integ-tests/src/test/java/com/google/integration/QuerylogsTest.java

```diff
@@ -0,0 +1,341 @@
+            getDoubleNotNull(rs, "NumResultOneMBRows"),
+            getDoubleNotNull(rs, "NumResultRows"),
+            getIntNotNull(rs, "NumSteps"),
+            getIntNotNull(rs, "NumStepswPar"),
+            getStringNotNull(rs, "ParamQuery"),
+            getDoubleNotNull(rs, "ParserCPUTime"),
+            getDoubleNotNull(rs, "ParserCPUTimeNorm"),
+            getDoubleNotNull(rs, "ParserExpReq"),
+            getLongNotNull(rs, "PersistentSpool"),
+            getBytesNotNull(rs, "ProcID"),
+            getStringNotNull(rs, "ProfileID"),
+            getStringNotNull(rs, "ProfileName"),
+            getStringNotNull(rs, "ProxyRole"),
+            getStringNotNull(rs, "ProxyUser"),
+            getStringNotNull(rs, "ProxyUserID"),
+            getStringNotNull(rs, "QueryBand"),
+            getBytesNotNull(rs, "QueryID"),
+            getStringNotNull(rs, "QueryRedriven"),
+            getStringNotNull(rs, "QueryText"),
+            getStringNotNull(rs, "ReDriveKind"),
+            getStringNotNull(rs, "RemoteQuery"),
+            getDoubleNotNull(rs, "ReqIOKB"),
+            getDoubleNotNull(rs, "ReqPhysIO"),
+            getDoubleNotNull(rs, "ReqPhysIOKB"),
+            getStringNotNull(rs, "RequestMode"),
+            getIntNotNull(rs, "RequestNum"),
+            getDoubleNotNull(rs, "SeqRespTime"),
+            getIntNotNull(rs, "SessionID"),
+            getStringNotNull(rs, "SessionTemporalQualifier"),
+            getLongNotNull(rs, "SpoolUsage"),
+            getTimestampNotNull(rs, "StartTime"),
+            getStringNotNull(rs, "StatementGroup"),
+            getIntNotNull(rs, "Statements"),
+            getStringNotNull(rs, "StatementType"),
+            getIntNotNull(rs, "SysDefNumMapAMPs"),
+            getIntNotNull(rs, "TacticalCPUException"),
+            getIntNotNull(rs, "TacticalIOException"),
+            getDoubleNotNull(rs, "TDWMEstMemUsage"),
+            getDoubleNotNull(rs, "ThrottleBypassed"),
+            getDoubleNotNull(rs, "TotalFirstRespTime"),
+            getDoubleNotNull(rs, "TotalIOCount"),
+            getLongNotNull(rs, "TotalServerByteCount"),
+            getStringNotNull(rs, "TTGranularity"),
+            getStringNotNull(rs, "TxnMode"),
+            getStringNotNull(rs, "TxnUniq"),
+            getStringNotNull(rs, "UnitySQL"),
+            getDoubleNotNull(rs, "UnityTime"),
+            getDoubleNotNull(rs, "UsedIota"),
+            getBytesNotNull(rs, "UserID"),
+            getStringNotNull(rs, "UserName"),
+            getIntNotNull(rs, "UtilityByteCount"),
+            getStringNotNull(rs, "UtilityInfoAvailable"),
+            getDoubleNotNull(rs, "UtilityRowCount"),
+            getDoubleNotNull(rs, "VHLogicalIO"),
+            getDoubleNotNull(rs, "VHLogicalIOKB"),
+            getDoubleNotNull(rs, "VHPhysIO"),
+            getDoubleNotNull(rs, "VHPhysIOKB"),
+            getStringNotNull(rs, "WarningOnly"),
+            getStringNotNull(rs, "WDName"));
```

Comment:
Yes, I know, it looks a bit ugly.
But the layout correlates with our other tests and it's as simple as possible.